Python/Product/Profiling/Profiling/StandaloneTargetView.cs

```diff
@@ -164,6 +164,9 @@ public StandaloneTargetView(IServiceProvider serviceProvider, StandaloneTarget t
             set {
                 if (_interpreterPath != value) {
                     _interpreterPath = value;
+                    if (Interpreter.Name == Strings.LaunchProfiling_OtherInterpreter && Interpreter.Path == null) {
+                        Interpreter = new PythonInterpreterView(Strings.LaunchProfiling_OtherInterpreter, "Global|"+ Strings.LaunchProfiling_OtherInterpreter, _interpreterPath);
```

Comment:
Where did this `Global|` string come from?

Reply:
I would consider making this special string combination a static readonly string somewhere, since you use it more than once.

Reply:
Good point, I used `Global` here because that's what all the other existing interpreters have in their names. Others also use it to search for corresponding `InterpreterFactoryProvider` here: https://github.com/microsoft/PTVS/blob/84add9c25ff26c1f39ab998400a8a66fb16ee60f/Python/Product/VSInterpreters/Interpreter/InterpreterRegistryService.cs#L304

But I can delete it since we are not searching by id.